### PR TITLE
Cleanup README.

### PR DESCRIPTION
**Pull request by mbforbes**
Not an overhaul or change of structure; mostly capitalization and indenting fixes, with a little tweaks for clarification

Preview at:
https://github.com/mbforbes/heapster/blob/readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,72 @@
...
+
+	```shell
+$ gcloud compute firewall-rules create monitoring-heapster --allow "tcp:8083" --target-tags=kubernetes-minion
+	```
+
+	Then, find out the IP address of the node where InfluxDB is running and point your web browser to `http://<nodeIP>:8083/`.
+
+	_Note: We are working on exposing the InfluxDB UI using the proxy service on the Kubernetes master._
+
+4. If you find InfluxDB to be using up a lot of CPU or memory, consider placing resource restrictions on the `InfluxDB & Grafana` pod. You can add `cpu: <millicores>` and `memory: <bytes>` in the [Controller Spec](deploy/influxdb-grafana-controller.yaml) and relaunch the controller:
```
Comment (vishh) [inline README.md]: nit: relaunch the controller's'

**Comment by vishh** [comment]
LGTM.

File: README.md (hunk 1 above)
Comment (mbforbes) [inline README.md]: Done.

**Comment by mbforbes** [comment]
Thanks for the quick review. I also fixed up a bad merge from the README update yesterday (that added port 8086 to the firewall rules).

**Comment by vishh** [comment]
Thanks for the PR @mbforbes :) . LGTM!

**Pull request merged**